### PR TITLE
update mx-puppet-bridge

### PR DESCRIPTION
**Pull request by lub**
I saw the brigde didn't get updated since two months, so I thought maybe it makes sense to bump the version. I also did some minor corrections along the way.

Seems to work so far.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Sorunome",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Sorunome/mx-puppet-bridge.git"
```
Comment (Sorunome) [inline package.json:16]: Wrong URL :)

File: package.json [hunk 2]
```diff
@@ -19,7 +24,7 @@
     "jimp": "^0.10.1",
     "js-yaml": "^3.13.1",
     "lodash": "^4.17.15",
-    "mx-puppet-bridge": "^0.0.39",
+    "mx-puppet-bridge": "^0.0.43",
```
Comment (Sorunome) [inline package.json:27]: newest version is 0.0.43-6

File: package.json (hunk 1 above)
Comment (lub) [reply package.json]: whoops :D

File: package.json (hunk 2 above)
Comment (lub) [reply package.json]: fixed it

**Comment by Sorunome** [comment]
Sorry for not checking back earlier, feel free to poke the hell out of soru in the matrix room! She manually applied these to not have to deal with merge conflicts

**Pull request closed without merging**